enterprise/frontend/src/embedding-sdk/components/private/PublicComponentWrapper/SdkError.tsx

```diff
@@ -2,20 +2,25 @@ import { t } from "ttag";
 
 import { useSdkSelector } from "embedding-sdk/store";
 import { getErrorComponent } from "embedding-sdk/store/selectors";
+import { Center } from "metabase/ui";
 
-export type SdkErrorProps = { message: string };
+export type SdkErrorProps = { message: string; className?: string };
 
-export const SdkError = ({ message }: SdkErrorProps) => {
+export const SdkError = ({ message, className }: SdkErrorProps) => {
   const CustomError = useSdkSelector(getErrorComponent);
 
   if (CustomError) {
     return <CustomError message={message} />;
   }
 
   return (
-    <div>
-      <div>{t`Error`}</div>
-      <div>{message}</div>
-    </div>
+    <Center className={className} h="100%" w="100%" mx="auto">
+      {CustomError || (
```

Comment:
you shouldn't need `CustomError` check here since it's already checked at line 12. At this point `CustomError` should be falsy.